logtail/logtail.go

```diff
@@ -433,7 +441,7 @@ func (l *Logger) awaitInternetUp(ctx context.Context) {
 // upload uploads body to the log server.
 // origlen indicates the pre-compression body length.
 // origlen of -1 indicates that the body is not compressed.
-func (l *Logger) upload(ctx context.Context, body []byte, origlen int) (uploaded bool, err error) {
+func (l *Logger) upload(ctx context.Context, body []byte, origlen int) (retryAfter int, err error) {
```

Comment:
totally up to you, but maybe return a `time.Duration` so the units are abundantly clear?

Reply:
Done.